### PR TITLE
feat: warp logo update

### PR DESCRIPTION
**Pull request by Grzesiu12**
Cannot run `yarn serve`
yarn-serve

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "mini-css-extract-plugin": "^1.6.0",
     "node-sass": "^6.0.0",
     "prismjs": "^1.26.0",
-    "redstone-api": "^0.4.7",
+    "redstone-api": "^0.4.11",
```
Comment (Grzesiu12) [inline package.json:28]: node-webcrypto failed on 0.4.7 (on ubuntu 22.04 using windows WSL2)
node-webcrypto

**Comment by ppedziwiatr** [comment]
@Grzesiu12 few notes:
1. the favicon also needs to be replaced
2. logo in mobile version also needs to be replaced

**Comment by Grzesiu12** [comment]
> 
done

**Comment by ppedziwiatr** [comment]
> Cannot run `yarn serve` yarn-serve

https://twitter.com/ppe_warp/status/1565336017317404673

**Pull request merged**